db/migrate/20230120172706_add_address_to_user.rb

```diff
@@ -1,6 +1,6 @@
 class AddAddressToUser < ActiveRecord::Migration[7.0]
-  def change
-    add_column :users, :address, :string
-    add_index :users, :address
-  end
+   def change
+      add_column :users, :address, :string
+      add_index :users, :address
+   end
```

Comment:
- [ ] The address column is not defined in the ERD, please 🙏🏽 remove it to have the correct database.